### DISCUSSION TITLE
Find renamed icons

### DISCUSSION THREAD

**Discussion by schonert** [lucide-icons/lucide#2165] [Q&A] [answered]
### Package

- [X] lucide
- [ ] lucide-angular
- [ ] lucide-flutter
- [ ] lucide-preact
- [X] lucide-react
- [ ] lucide-react-native
- [ ] lucide-solid
- [ ] lucide-svelte
- [ ] lucide-vue
- [ ] lucide-vue-next
- [ ] Figma plugin
- [ ] source/main
- [ ] other/not relevant

### Version

@latest

### Can you reproduce this in the latest version?

- [ ] Yes
- [ ] No

### Browser

- [ ] Chrome/Chromium
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] iOS Safari
- [ ] Opera
- [ ] Other/not relevant

### Operating system

- [ ] Windows
- [ ] Linux
- [X] macOS
- [ ] ChromeOS
- [ ] iOS
- [ ] Android
- [ ] Other/not relevant

### Description

Love this project! I noticed that occasionally icons are renamed. What's the best way to get an overview of renamed icons when upgrading the package? 

### Steps to reproduce

NPM install

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)

**Comment by karsa-mistmere** [comment, marked as answer]
@schonert: We always add backwards compatible aliases, you normally don't have to worry about this as long as your package is up to date. The most common issue with renames is that people use outdated packages and can't fathom why the names aren't consistent with lucide.dev, which, of course, uses the latest names.

As of #1960 (to be released in v0.380.0) these backwards compatible aliases will also occasionally have a deprecation notice, any renames that ought to be addressed will then be marked for you.
